components/uid-input/src/uid-input.ts

```diff
@@ -1,3 +1,3 @@
-import { UidInput } from './UidInput.js';
+import { UidInput } from './UidInput';
```

Comment:
:question: Is it not possible to delete this file and put the code somewhere (in index.ts directly or in uidInput.ts). Same question for uid-text